dotCMS/src/integration-test/java/com/dotcms/util/FiltersUtil.java

```diff
@@ -50,7 +51,7 @@ private FiltersUtil() {
     /**
      * Creates a new Vanity URL contentlet
      */
-    public Contentlet createVanityUrl(String title, String site, String uri,
+    public DefaultVanityUrl createVanityUrl(String title, String site, String uri,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'uri' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5434315)